docs/Zendesk-batchsource.md

```diff
@@ -39,7 +38,7 @@ or equal to the specified date. The date must be provided in the date format:
 
 Start Date is required for batch objects like: Ticket Comments, Organizations, Ticket Metric Events, Tickets, Users.
 
-**End Date:** Filter data to include only records which have Zendesk modified date is less than
+**End Date:** Filter data to include only records which have Zendesk modified date that is less than
```

Comment:
Here's what's in the doc: Filter data to include only records that have a Zendesk modified date that is less than the specified date.

Reply:
updated